--- a/src/wsh/scripts/md5.wsh
+++ b/src/wsh/scripts/md5.wsh
@@ -1,4 +1,4 @@
-#!/home/jonathan/solution-exp/unlinking/awareness/self/wcc/src/wsh/wsh
+#!/usr/bin/wsh
 
 -- Computing a MD5 sum using cryptographic functions from foreign binaries (eg: sshd/OpenSSL)
 
--- a/src/wsh/scripts/read.wsh
+++ b/src/wsh/scripts/read.wsh
@@ -1,4 +1,4 @@
-#!/home/jonathan/solution-exp/unlinking/awareness/self/wcc/src/wsh/wsh
+#!/usr/bin/wsh
 
  fname="/etc/passwd"
  printf("\n ** Reading file %s\n", fname)
